web-wallet/src/lib/components/GasControls/GasControls.svelte

```diff
@@ -79,6 +79,7 @@
 
 	onMount(() => {
 		checkGasLimits();
+		dispatchGasChange();
```

Comment:
Why do we need to dispatch a change `onMount`?

Reply:
Because I was not getting the default values from the settings store I had to dispatch them when the `GasControls` component was mounted.